README.md

```diff
@@ -2,31 +2,818 @@
+</td>
+<td>
+
+```text
+name        breed
+
+Snuggles    Bichon
+Honey       Cavachon
+```
+
+</td>
+</tr>
+
+<tr>
+<td>
+The IN keyword can also be
+used to match a list of possible values:
+
+```sql
+SELECT name, breed
+FROM pet
+WHERE breed IN ( 'Bichon' , 'Cavachon' );
+```
+
+</td>
+<td>
+
+```text
+name        breed
+
+Snuggles    Bichon
+Honey       Cavachon
+```
+
+</td>
+</tr>
+
+
+
+
+<tr>
+<td>
+The NOT keyword retrieves all rows that
+do not match the predicate:
+
+```sql
+SELECT name, species
+FROM pet
+WHERE species NOT IN ( 'dog' , 'cat' );
+```
+
+</td>
+<td>
+
+```text
+name    species
+
+Herbie  fish
+```
+
+</td>
+</tr>
+
+
+
+<tr>
+<td>
+The NULL keyword is used to
+test for missing or unknown values:
+
+```sql
+SELECT id, name, age
+FROM pet
+WHERE age IS NULL
+```
+
+</td>
+<td>
+
+```text
+id  name    age
+
+5   Honey   null
+```
+
+</td>
+</tr>
+
+
+<tr>
+<td>
+Use NOT NULL to
+test for known values:
+
+```sql
+SELECT id, name, age
+FROM pet
+WHERE age IS NOT NULL
+```
+
+</td>
+<td>
+
+```text
+id  name        age
+1   Moe         10
+2   Hana        1
+3   Lil' Bub    5
+4   Snuggles    5
+6   Herbie      4
+7   Fifi        1
+8   Lulu        12   
+```
+
+</td>
+</tr>
+
+</table>
+
+
+
+### Pattern Matching
+
+
+The `LIKE` keyword is used in a WHERE clause to look for a pattern
+in a column.  We can use two wildcards with LIKE:
+
+- `%`  Represents zero, one, or multiple characters   
+- `_`  Represents a single character
+
+
+For example:
+
+- `'%abc'` matches words that end in "abc".
+- `'abc%'` matches words that start with "abc".
+- `'%abc%'` matches words that contain "abc" as a substring.
+- `'a_b'` matches 3 letter words that start with "a" and end with "b".
+
+We can also use functions that convert a string to lowercase or
+uppercase to perform case-insensitive matching:
+
+- `lower(str)` - return the lowercase version of string `str`.
+- `upper(str)` - return the uppercase version of string `str`.
+
+<table>
+<tr>
+<th>
+Query
+</th>
+<th>
+Result
+</th>
+</tr>
+
+<tr>
+<td>
+Display name and breed
+where breed ends in "chon":
+
+```sql
+SELECT name, breed
+FROM pet
+WHERE breed LIKE '%chon';
+```
+
+</td>
+<td>
+
+```text
+name        breed
+
+Snuggles    Bichon
+Honey       Cavachon
+```
+
+</td>
+</tr>
+
+
+<tr>
+<td>
+Display name and breed
+where breed contains the
+substring "poo" (case sensitive).
+
+
+```sql
+SELECT name, breed
+FROM pet
+WHERE breed LIKE '%poo%';
+```
+
+</td>
+<td>
+
+```text
+name   breed
+
+Lulu    Yorkiepoo
+```
+
+</td>
+</tr>
+
+<tr>
+<td>
+Display name and breed
+where breed contains the
+substring "poo" (case insensitive match).
+
+```sql
+SELECT name, breed
+FROM pet
+WHERE lower(breed) LIKE '%poo%';
+```
+
+</td>
+<td>
+
+```text
+name   breed
+
+Fifi    Poodle
+Lulu    Yorkiepoo
+```
+
+</td>
+</tr>
+
+</table>
+
+
+
+## ORDER BY
+
+The `ORDER BY` clause sorts the retrieved rows by order of one or more columns.
+The optional keywords `asc` and `desc` will sort respectively
```

Comment:
I usually uppercase ASC and DESC to show that it is a SQL operation. It looks like you capitalized them in the learning goals. For consistency, can we either change the casing here or in the learning goals please?

Reply:
thanks for catching this.  queries fixed to capitalize.